fusequery/query/src/configs/config.rs

```diff
@@ -135,16 +137,68 @@ pub struct Config {
 
     #[structopt(long, env = STORE_API_ADDRESS, default_value = "127.0.0.1:9191")]
     pub store_api_address: String,
+ 
+    #[structopt(long, env = "STORE_API_USERNAME", default_value = "root")]
+    pub store_api_username: User,
 
-    #[structopt(long, env = STORE_API_USERNAME, default_value = "root")]
-    pub store_api_username: String,
+    #[structopt(long, env = "STORE_API_PASSWORD", default_value = "root")]
+    pub store_api_password: Password,
 
-    #[structopt(long, env = STORE_API_PASSWORD, default_value = "root")]
-    pub store_api_password: String,
 
     #[structopt(long, short = "c", env = CONFIG_FILE, default_value = "")]
     pub config_file: String,
 }
+#[derive(Clone, serde::Deserialize, PartialEq, StructOpt, StructOptToml)]
+#[serde(default)]
+pub struct Password{
+   pub store_api_password: String,
+}
+
+impl AsRef<String> for Password {
+    fn as_ref(&self) -> &String{
+        &self.store_api_password
+    }
+}
+
+impl FromStr for Password{
+    type Err = ErrorCode;
+    fn from_str(s: &str) -> common_exception::Result<Self> {
+        Ok(Self{
+            store_api_password: s.to_string(),
+        })
+    }
+}
+
+impl fmt::Debug for Password{
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "******")
+    }
+}
+#[derive(Clone, serde::Deserialize, PartialEq, StructOpt, StructOptToml)]
```

Comment:
need new line

Reply:
Do we have some tool or extension (in the Vscode) can do this formatting thing automatically? Give a suggestion if have. Thanks.

Reply:
![image](https://user-images.githubusercontent.com/7311971/123929365-58671c00-d9c1-11eb-999f-acba26a26d5d.png)
This Rust Assist extesion looks good. Do we need this format tool？

Reply:
We use rust-fmt to format the code. You can use `make lint` to check the format